app/application/services/email_notification_service.py

```
@@ -0,0 +1,81 @@
+        """
+        from app.infrastructure.notifications.email_metrics import (
+            increment_email_sent,
+            increment_email_failed,
+        )
+
+        try:
+            await self._email_service.send_otp(to_email, otp)
+            await increment_email_sent()
+        except Exception as e:
+            await increment_email_failed()
+            log.error("Failed to send OTP email", extra={"error": str(e), "to": to_email})
```

Ruff (0.15.1)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)